pkgs/by-name/no/notion-app/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
+      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "notion-app";
+
+  version = info.version;
+
+  meta = {
```

Comment:
`meta` is the last part

Reply:
Is there any benefit to moving meta to the end?

Reply:
That's what all packages do since before I joined here in 2021.